components/bsx/Create/CreateToken.vue

```diff
@@ -147,23 +152,23 @@ export default class CreateToken extends mixins(
 ) {
   @Prop({ type: Boolean, default: false }) showExplainerText!: boolean
 
-  protected base: BaseTokenType<MintedCollection> = {
+  public base: BaseTokenType<MintedCollection> = {
```

Comment:
Why change of visibility?

Reply:
linter not happy
`Property 'base' is protected and only accessible within class 'CreateToken' and its subclasses`

Reply:
🫡 Hvala!